testing/endpoint.go

```diff
@@ -473,26 +475,35 @@ func (endpoint *Endpoint) ChanCloseInit() error {
 // The packet sequence generated for the packet to be sent is returned. An error
 // is returned if one occurs.
 func (endpoint *Endpoint) SendPacketV2(
```

Comment:
this functionality exists in this fn https://github.com/cosmos/ibc-go/blob/1682792b9428df0df33709b8192c8bae3b508f71/testing/endpoint_v2.go#L11